### DISCUSSION TITLE
Is there any way to SEE binary contents for image compare?

### DISCUSSION THREAD

**Discussion by nanakinoir** [WinMerge/winimerge#25] [Q&A] [answered]
I'm trying to open up the hex editor (per https://manual.winmerge.org/en/Compare_bin.html) when comparing two .jpg files, but it keeps opening the visual comparison. I've tried everything I could think of:

1. removing .jpg extension from Options->Compare->Image list
![image](https://user-images.githubusercontent.com/19397881/135772090-89d8547e-0e12-4f7a-8bb9-87e33393c20e.png)

2. adding .jpg extension to Options->Compare->Binary list
![image](https://user-images.githubusercontent.com/19397881/135772108-ec6321b3-5274-44f4-a173-08418b551978.png)

3. renaming files to .bin
![image](https://user-images.githubusercontent.com/19397881/135772306-f947e289-f1f8-45f3-8521-915ee903f182.png)

As you can see, I'm still getting the visual comparator even if I renamed the files. How do I get to the underlying binary/hex comparison?

**Comment by sdottaka** [comment, marked as answer]
If you leave the Image File Patterns empty, the default extension pattern list will be used. (This may change in the future).
For this reason, please specify an arbitrary extension pattern that does not exist, such as "*.aaaa".

**Reply by nanakinoir** [reply]
Thanks! It works.
